README.md

```diff
@@ -413,6 +384,36 @@ With this rule set, the following rules will apply:
 
 There are filters in place that can be applied to change the behavior for what's allowed and what's not allowed.
 
+### `vip_governance__governance_file_path`
+
+Change the governance rules file that's used by the plugin, based on a variety of filter options that are available. By default, it is set to
+the path to `governance-rules.json` in the private directory in a VIP site. For non-vip sites, it is set to the path to `governance-rules.json` in the plugin directory.
```

Comment:
Wouldn't it be better to say it checks first for the private directory and then to the plugin directory?

Reply:
Unrelated nit, was the newline between lines `389` and `390` on purpose?

Reply:
Good catch, that should not be the case.